auto_inertial_params/proposal.md

```diff
@@ -87,30 +87,32 @@ The `<inertia>` tag, on the other hand, needs to enclose the following 6 tags:
  * `<izz>` (Moment of Inertia)
 
  * `<ixy>` (Product of Inertia)
- 
+
  * `<ixz>` (Product of Inertia)
- 
+
  * `<izy>` (Product of Inertia)
 
-This proposal suggests the addition of the following elements and attributes in `SDFormat Spec`: 
-
- 1.  `//inertial/@auto` attribute that would tell `libsdformat` to calculate the Inertial 
- values (Mass, Mass Matrix & Inertial Pose) automatically for the respective link. 
-
- 2. `//collision/density` element for specifying the density of the collision geometry of a link. 
- This density values would be used to calculate the inertial parameters of the respective collision 
- geometries. Adding this as part of the `<collision>` tag would allow a user to simulate links with 
- different material types for different collisions. By default, the value of density would be set 
- equal to that of water which is 1000 kg/m^3. A `//link/inertial/density` element would also be 
- added in the spec to allow users to specify the density values on a link level instead of 
- specifying the same values for each collision.
-
- 3. `//collision/auto_inertia_params` element would be added that can be used to provide some 
- parameters or options for a custom inertia calculator. Similar to the `density` element above, 
- a link-level `//link/inertial/auto_inertia_params` element would also be added. This would allow 
- the user to provide inertia calculator parameters for the whole link while retaining the 
- flexibility to specify different parameters for each collision. Custom elements and attributes 
- using a namespace prefix can be used to provide the user-defined parameters for a custom calculator. 
+This proposal suggests the addition of the following elements and attributes in `SDFormat Spec`:
+
+ 1.  `//inertial/@auto` attribute that would tell `libsdformat` to calculate the Inertial
+ values (Mass, Mass Matrix & Inertial Pose) automatically for the respective link.
+
+ 2. `//collision/density` element for specifying the density of the collision geometry of a link.
+ This density values would be used to calculate the inertial parameters of the respective collision
+ geometries. Adding this as part of the `<collision>` tag would allow a user to simulate links with
+ different material types for different collisions. By default, the value of density would be set
+ equal to that of water which is 1000 kg/m^3. A `//link/inertial/density` element would also be
+ added in the spec to allow users to specify the density values on a link level instead of
+ specifying the same values for each collision. Note that if `//link/inertial/mass` is also
+ specified, the link's inertial parameters will be scaled to match the input mass while
+ respecting the ratio of collision density values.
```

Comment:
this looks good to me; I think we just need to note which versions of libsdformat support this change (presumably it will be in 15.2 and 14.7)

Reply:
added libSdformat versions to the note. [d405d34](https://github.com/gazebosim/sdf_tutorials/pull/100/commits/d405d34870ca8feae6469fed69844b90157df982)